smoke-tests/images/early-jdk8/build.gradle.kts

```diff
@@ -18,18 +18,20 @@ tasks {
     from("Dockerfile")
   }
 
+  val repo = System.getenv("GITHUB_REPOSITORY") ?: "open-telemetry/opentelemetry-java-instrumentation"
```

Comment:
defaulting isn't needed in CI, but added in case someone is testing publishing locally with their own credentials(?) or maybe that's not good and I should just not default?

Reply:
I realized these tags are also used when just building local docker image, so makes sense to keep the defaults I think